apps/client-server/src/app/form-generator/form-generator.service.ts

```diff
@@ -34,7 +34,7 @@ export class FormGeneratorService {
    */
   async generateForm(
     request: FormGenerationRequestDto
-  ): Promise<FormBuilderMetadata<never>> {
+  ): Promise<FormBuilderMetadata> {
```

Comment:
I've removed a lot of generics from FormBuilderMetadata and from the AggregateFieldType because they were not used anyway

You still can specify the type to use defaultFrom tho, just add it into the decorator like this:
```ts
const someObject = { someKey: false }
class TestType {
   @BooleanField<typeof someObject>({ defaultFrom: 'someKey' })
   someBoolean = true
}
```